ChaosMod/Effects/db/Misc/MiscQuickSprunkStop.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+    Effect by Last0xygen
+*/
+
+#include <stdafx.h>
+
+
+static void OnStart()
+{
+    static Hash objectHash = GET_HASH_KEY("prop_vend_soda_02");
+    Ped player = PLAYER_PED_ID();
+    Vector3 pos = GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS(player, 0, 5, 0);
+    Object obj = CreatePoolProp(objectHash, pos.x, pos.y, pos.z, true);
```

Comment:
Objects which are marked as not needed afterwards should be created through the corresponding native instead